services/email.py

```
@@ -0,0 +1,81 @@
+                format='full'
+            ).execute()
+            
+            if 'payload' in message:
+                parts = message['payload'].get('parts', [])
+                for part in parts:
+                    if part['mimeType'] == 'text/plain':
+                        data = part['body'].get('data', '')
+                        return base64.urlsafe_b64decode(data).decode('utf-8')
+            return ''
+        except Exception as e:
+            raise Exception(f"Error getting email content: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)